ci/release/Dockerfile

```diff
@@ -3,6 +3,9 @@ FROM debian:latest
 
 ARG TARGETARCH
 
+RUN apt-get update && \
```

Comment:
It is better to add these lines before `ARG TARGETARCH` to use a cache for several architectures

Reply:
I don't believe that's correct. The cache cannot be shared between architectures as the base images will be different.

Reply:
The docker image is layered) before "copying" d2 into docker the image is the same. And ca-certificates may be cached.

Reply:
I don't follow you. When the architecture is changed, `debian:latest` refers to a different image. Thus there is no shared cache between architectures.

Reply:
@pleshevskiy here is a reference in case you weren't aware of multi arch builds
https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/

we call `docker buildx` in `build.sh`